CHANGES.txt

```diff
@@ -60,6 +60,9 @@ Breaking Changes
 Changes
 =======
 
+- Introduced a new VIEW privilege class which is used when granting/denying
+  access on views.
```

Comment:
Maybe consider extending the existing view entry https://github.com/crate/crate/pull/7140/files#diff-bb3c4deeba77c2bbfd893f2e90fecc77R74 instead. Otherwise if you read this from top to bottom we mention view privileges without the introduction of views.